packages/react/src/index.js

```diff
@@ -14,6 +14,35 @@ import { Box } from './components/Box';
 import useComponentOverrides from './theme/useComponentOverrides';
 import { ToastBarProvider } from './components/ToastBar';
 
+// Include scrollbar styles directly in JavaScript
+const scrollbarStyles = `
+  /* Style the main scrollbar */
+  ::-webkit-scrollbar {
+    width: 12px;
+  }
+
+  /* Style the scrollbar thumb (the draggable part) */
+  ::-webkit-scrollbar-thumb {
+    background: #7f7f7f;
+    border-radius: 12px;
+  }
+
+  /* Style the scrollbar thumb on hover */
+  ::-webkit-scrollbar-thumb:hover {
+    background: #555555;
+  }
+
+  /* Style the scrollbar track (the background) */
+  ::-webkit-scrollbar-track {
+    background: #ffffff;
+  }
+`;
+
+const style = document.createElement('style');
+style.type = 'text/css';
+style.innerHTML = scrollbarStyles;
+document.getElementsByTagName('head')[0].appendChild(style);
```

Comment:
When creating an NPM package that includes React components, it's crucial to provide styles in a way that they are encapsulated within the components themselves and do not unintentionally affect the global scope of the consuming application.